packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const ignoredOperations = [
+  /* missing route */
+  /* #5694 */
+  "getSyncCommitteeRewards",
+  "getBlockRewards",
+  "getAttestationsRewards",
+  "getDepositSnapshot", // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  "getBlindedBlock", // #5699
+  "getNextWithdrawals", // #5696
+  "getDebugForkChoice", // #5700
+  /* #6080 */
+  "getLightClientBootstrap",
+  "getLightClientUpdatesByRange",
+  "getLightClientFinalityUpdate",
+  "getLightClientOptimisticUpdate",
+  "getPoolBLSToExecutionChanges",
+  "submitPoolBLSToExecutionChange",
+];
+
+const ignoredProperties: Record<string, IgnoredProperty> = {
+  /*
+   #5693
+   missing finalized
+   */
+  getStateRoot: {responseProperties: ["finalized"]},
```

Comment:
I like the nested property approach, we anyways don't have type safety for the props so string dot notation is perfectly fine I think and semantics are nice

You could even go a step further if you wann to get rid of the objects here altogether

```suggestion
  getStateRoot: ["response.finalized"],
```

or I guess to make it less verbose could also just use "response" as properties is already implied here

```suggestion
  getStateRoot: {response: ["finalized"]},
```

I guess with 2nd approach (currently used) we get type safety on request / response properties which is kinda good

Reply:
2nd option looks good!